main/SQLHelper.cpp

```diff
@@ -2427,6 +2427,7 @@ bool CSQLHelper::OpenDatabase()
 		{
 			safe_query("UPDATE Hardware SET Mode1 = 5000 WHERE Type = %d", HTYPE_DenkoviSmartdenIPInOut);
 			safe_query("UPDATE Hardware SET Mode1 = 5000 WHERE Type = %d", HTYPE_DenkoviSmartdenLan);
+			safe_query("UPDATE Hardware SET Mode1 = 5000 WHERE Type = %d", HTYPE_DenkoviDevices);
```

Comment:
You need to increase the database version (to 130 as we speak) and make a new 'if' line for this change

Reply:
Thanks for the remind. Is this mean to remove this line 
```
			safe_query("UPDATE Hardware SET Mode1 = 5000 WHERE Type = %d", HTYPE_DenkoviDevices);
```
and add
```
if (dbversion < 130)
		{
			safe_query("UPDATE Hardware SET Mode1 = 5000 WHERE Type = %d", HTYPE_DenkoviDevices);
		}
```
Thanks in advance.

Reply:
yes it is, thanks !